docker/0.90-2/resources/mms/ExecutionParameters.java

```diff
@@ -0,0 +1,86 @@
+import com.google.gson.GsonBuilder;
+import com.google.gson.annotations.SerializedName;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.Properties;
+import software.amazon.ai.mms.servingsdk.Context;
+import software.amazon.ai.mms.servingsdk.ModelServerEndpoint;
+import software.amazon.ai.mms.servingsdk.annotations.Endpoint;
+import software.amazon.ai.mms.servingsdk.annotations.helpers.EndpointTypes;
+import software.amazon.ai.mms.servingsdk.http.Request;
+import software.amazon.ai.mms.servingsdk.http.Response;
+
+// The modified endpoint source code for the jar used in this container.
+// You can create this endpoint by moving it by cloning the MMS repo:
+// > git clone https://github.com/awslabs/mxnet-model-server.git
+//
+// Copy this file into plugins/endpoints/src/main/java/software/amazon/ai/mms/plugins/endpoints/
+// and then from the plugins directory, run:
+//
+// > ./gradlew fJ
+//
+// The jar should be available in plugins/build/libs as endpoints-1.0.jar
+@Endpoint(
+        urlPattern = "execution-parameters",
+        endpointType = EndpointTypes.INFERENCE,
+        description = "Execution parameters endpoint")
+public class ExecutionParameters extends ModelServerEndpoint {
+
+    @Override
+    public void doGet(Request req, Response rsp, Context ctx) throws IOException {
+        Properties prop = ctx.getConfig();
+        SagemakerXgboostResponse r = new SagemakerXgboostResponse();
+        r.setMaxConcurrentTransforms(Integer.parseInt(prop.getProperty("NUM_WORKERS", "1")));
+        r.setBatchStrategy("MULTI_RECORD");
+        r.setMaxPayloadInMB(Integer.parseInt(prop.getProperty("SAGEMAKER_MAX_REQUEST_SIZE", "6")));
```

Comment:
SAGEMAKER_MAX_REQUEST_SIZE is a value in bytes. I think you need to divide by 1024*1024

Reply:
Yeap, that's what I was changing, and i'm having trouble building the changes right now.